posts/01-first-map/ZimmZone/tvhs.md

```diff
@@ -1,4 +1,4 @@
 # WELCOME TO THE HOME OF THE EAGLES!
 
-Located in Loveland, Colorado. Thompson Valley High School was built in 1975 and has a vision to provide a profoundly inclusive and diverse learning environment that meets students where they are and empowers them to achieve their personal and academic goals, allowing them to support their life aspirations.
+Located in Loveland, Colorado. [Thompson Valley High School](https://tvhs.tsd.org/about) was built in 1975 and has a vision to provide a profoundly inclusive and diverse learning environment that meets students where they are and empowers them to achieve their personal and academic goals, allowing them to support their life aspirations.
 <embed type="text/html" src="tvhs1.html" width="600" height="600">
```

Comment:
Make sure to add the tvhs1.html file to the posts/01-first-map/ZimmZone folder for your embedding